app/main.py

```diff
@@ -1,3 +1,50 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.service import (fuel_data,
```

Comment:
after the opening bracket, write imports from a new line, and write the closing bracket from a new line